build.gradle

```diff
@@ -1779,15 +1790,16 @@ void addMavenPublication(Project project, List<String> projectDependencies) {
 
                         Node projectDependencyPlatform = dependencies.appendNode("dependency")
                         projectDependencyPlatform.appendNode("groupId", MAVEN_GROUP_ID)
-                        projectDependencyPlatform.appendNode("artifactId", "javafx-${project.name}")
+                        projectDependencyPlatform.appendNode("artifactId", artifactName)
                         projectDependencyPlatform.appendNode("version", MAVEN_VERSION)
                         projectDependencyPlatform.appendNode("classifier", "\${javafx.platform}")
 
                         if (!projectDependencies.empty) {
                             projectDependencies.each { dep ->
+                                def depName = dep.moduleName.replace('.', '-')
```

Comment:
I was doing one last check and noticed a bug here. As with the earlier block on line 1716, the `projectDependencies.each` line needs to be replaced with:

```
    projectDependencies.each { projName ->
        def dep = project.project(":$projName")
```

since `dep` needs to be a gradle Project not a String.

I'll test it and fix it.

Reply:
Fixed.